docs/source/host/server-setup/linux-server.md

```diff
@@ -6,11 +6,11 @@ The Overte packages can help you get your own domain up and running quickly.
 
 You can run these same commands on an existing Overte domain to upgrade it if the original domain was installed using the package. Packages are currently available for the following distributions:
 
-### Ubuntu Server 18.04 LTS
+### Ubuntu Server 22.04 LTS
```

Comment:
We don't just support Ubuntu 22.04, but also Ubuntu 20.04, Ubuntu 18.04, Debian 11, Fedora 36, Fedora 37, and Rocky Linux 9.
For Ubuntu 22.04, Debian 11, Fedora 36, and Fedora 37 we also support aarch64.